frontend/src/scenes/insights/InsightDateFilter/insightDateFilterLogic.test.ts

```diff
@@ -0,0 +1,45 @@
+import { initKeaTestLogic } from '~/test/init'
+import { router } from 'kea-router'
+import { expectLogic } from 'kea-test-utils'
+import { insightDateFilterLogic } from 'scenes/insights/InsightDateFilter/insightDateFilterLogic'
+
+const insightsURL = (
+    additionalPathPart: string = '',
+    dateFrom?: string | undefined,
+    dateTo?: string | undefined
+): string => {
+    let url = `/insights/smosHASp${additionalPathPart}?insight=TRENDS&interval=day&actions=%5B%5D&events=%5B%7B"id"%3A"%24pageview"%2C"name"%3A"%24pageview"%2C"type"%3A"events"%2C"order"%3A0%7D%5D&properties=%5B%5D&filter_test_accounts=false&display=ActionsPie`
```

Comment:
How bout using `urls.insightView()`? Would handle all the filters too – otherwise it'll be much easier for changes in insight routing to break this